lib/shared/permissions.js

```diff
@@ -198,6 +198,13 @@ exports.elevateCommand = (command, options) => {
   // for now, we should make sure we double check if the error messages
   // have changed every time we upgrade `sudo-prompt`.
 
+  }).catch((error) => {
+    return _.includes(error.message, 'is not in the sudoers file');
+  }, () => {
+    throw errors.createUserError({
+      title: 'Your user doesn\'t have enough permissions to proceed',
+      description: 'This application requires sudo permissions to be able to write to drives'
```

Comment:
Nit: I think these lines should be referring to 'privileges' rather than 'permissions'
(there was something about the phrasing that was niggling me, and I just worked out what it was ;)  )